FerramAerospaceResearch/RealChuteLite/RealChuteFAR.cs

```diff
@@ -443,21 +443,21 @@ public void AssumeDragCubePosition(string cubeName)
                 case "STOWED":
                 {
                     parachute.gameObject.SetActive(false);
-                    cap.gameObject.SetActive(true);
+                    if (cap != null) cap.gameObject.SetActive(true);
```

Comment:
I don't think any of the `cap != null` checks are needed here, `cap == null` shows a bad installation that should be fixed instead

Reply:
For parachutes on EVA kerbals in command seats, the `cap = part.FindModelTransform(capName);` on line 880 of RealChuteFAR.cs returns `null`. This is possibly due to a local MM patch that renames `ModuleEvaChute` as `RealChuteFAR` (which is possibly not the best). In any case, if the `cap` transform isn't found for some reason, then the `if (cap != null)` checks need to be made to avoid NREs.

Reply:
In that case, `cap?.gameObject` would be more appropriate and consice as `cap` is never a valid Unity object in the first place.

Reply:
Yes, that'd be better.